neonvm/runner/main.go

```diff
@@ -527,11 +531,46 @@ func main() {
 		qemuCmd = append(qemuCmd, "-incoming", fmt.Sprintf("tcp:0:%d", vmv1.MigrationPort))
 	}
 
+	go terminateQemu(vmSpec.QMP)
```

Comment:
Is this required? If not, it'd be good to have this as a separate PR

Reply:
@sharnoff it's just graceful shutdown for VM. What objections do not add it in this PR ?

Reply:
The main problem I have is that, without reading the content of this PR, I had no idea that it implemented the graceful shutdown feature. I tend to split features into their own PRs (sometimes too much), but a reasonable alternative IMO is to just make sure that the features are included in the PR/commit name.

As an example, this was the motivation for #146. Here is a more significant case because there's actually a whole separate feature.

Anyways - do what you think is right. I'd _prefer_ to either have graceful shutdown in the commit message or as a separate PR, but it's up to you.

Reply:
Fair enough, go for it